package.json

```diff
@@ -17,16 +17,16 @@
   "dependencies": {
     "bluebird": "3.4.1",
     "byline": "4.2.1",
-    "caw": "1.2.0",
+    "caw": "2.0.0",
     "chalk": "1.1.3",
     "commondir": "1.0.1",
     "debug": "2.2.0",
-    "got": "5.4.1",
+    "got": "6.3.0",
```

Comment:
Sorry, we can't update got because it's not going to work with Node 0.12. (or was it 0.10?) Same with caw.